src/azure-cli/azure/cli/command_modules/acr/connected_registry.py

```diff
@@ -0,0 +1,494 @@
+        add_client_token_set = set(add_client_token_list)
+    else:
+        add_client_token_set = set()
+    if remove_client_token_list is not None:
+        for i, client_token_name in enumerate(remove_client_token_list):
+            remove_client_token_list[i] = build_token_id(
+                subscription_id, resource_group_name, registry_name, client_token_name)
+        remove_client_token_set = set(remove_client_token_list)
+    else:
+        remove_client_token_set = set()
+
+    duplicate_client_token = set.intersection(add_client_token_set, remove_client_token_set)
+    if duplicate_client_token:
+        errors = sorted(map(lambda action: action[action.find('/') + 1:], duplicate_client_token))
+        raise CLIError(
+            'Update ambiguity. Duplicate client token ids were provided with ' +
+            '--add-client-tokens and --remove-client-tokens arguments.\n{}'.format(errors))
+
+    current_client_token_set = set(current_connected_registry.client_token_ids) \
+        if current_connected_registry.client_token_ids else set()
+    client_token_set = current_client_token_set.union(add_client_token_set).difference(remove_client_token_set)
+
+    client_token_list = list(client_token_set) if client_token_set != current_client_token_set else None
+
+    ConnectedRegistryUpdateParameters, SyncUpdateProperties, LoggingProperties = cmd.get_models(
+                'ConnectedRegistryUpdateParameters', 'SyncUpdateProperties', 'LoggingProperties')
+    connected_registry_update_parameters = ConnectedRegistryUpdateParameters(
+        sync_properties=SyncUpdateProperties(
+            token_id=current_connected_registry.parent.sync_properties.token_id,
+            schedule=sync_schedule,
+            message_ttl=sync_message_ttl,
+            sync_window=sync_window
+        ),
+        logging=LoggingProperties(
+            log_level=log_level,
+            audit_log_status=sync_audit_logs_enabled
+        ),
+        client_token_ids=client_token_list
+    )
+
+    try:
+        return client.update(resource_group_name=resource_group_name,
+                             registry_name=registry_name,
+                             connected_registry_name=connected_registry_name,
+                             connected_registry_update_parameters=connected_registry_update_parameters)
+    except ValidationError as e:
+        raise CLIError(e)
+
+
+def acr_connected_registry_delete(cmd,
+                                  client,
+                                  connected_registry_name,
+                                  registry_name,
+                                  yes=False,
+                                  resource_group_name=None):
+
+    _, resource_group_name = validate_managed_registry(
+        cmd, registry_name, resource_group_name)
+
+    user_confirmation("Are you sure you want to delete the Connected Registry '{}' from '{}'?".format(
```

Comment:
>Connected Registry [](start = 59, length = 18)

keep the message format consistent in the code `Connected Registry` => `connected registry`